toolchain/language_server/handle_text_document.cpp

```diff
@@ -35,17 +103,12 @@ auto HandleDidChangeTextDocument(
     return;
   }
 
-  // Full text is sent if full sync is specified in capabilities.
-  if (params.contentChanges.size() != 1) {
-    CARBON_DIAGNOSTIC(LanguageServerUnsupportedChanges, Warning,
-                      "received unsupported contentChanges count: {0}", int);
-    context.file_emitter().Emit(filename, LanguageServerUnsupportedChanges,
-                                params.contentChanges.size());
-    return;
-  }
   if (auto* file = context.LookupFile(filename)) {
-    file->SetText(context, params.textDocument.version,
-                  params.contentChanges[0].text);
+    // we copy the document to a new string, apply changes to the string and
+    // set the string as the new text
```

Comment:
```suggestion
    // We copy the document to a new string, apply changes to the string, and
    // set the string as the new text.
```

Capitalization/punctuation.